package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "@babel/runtime": "7.15.4",
     "@patternfly/patternfly": "4.132.2",
-    "@patternfly/react-core": "4.152.4",
+    "@patternfly/react-core": "4.198.19",
```

Comment:
Is this bump required to get the proper compact styling? If so, we should also bump any other `@patternfly` packages as well to the latest version.

Reply:
Yeah I had to update to this version to get compact labels

Reply:
Then lets bump all of the `@patternfly` packages to the latest version.

Reply:
Should I have the "^" in the version to ensure only minor versions are installed?

Reply:
@AsToNlele I wouldn't add any version modifiers to Patternfly, because it can change design and layout of the app not only during minor releases but sometimes even during patch version increments.

Reply:
Thanks, will do!